packages/vaadin-grid/src/vaadin-grid.js

```diff
@@ -964,8 +964,8 @@ class GridElement extends ElementMixin(
   }
 
   /** @protected */
-  __updateVisibleRows() {
-    this.__virtualizer && this.__virtualizer.update();
+  __updateVisibleRows(start, end) {
+    this.__virtualizer && this.__virtualizer.update(start, end);
```

Comment:
Could you provide a reference to where it is called with `start`, `end` arguments?

My first guess was that the arguments are used here, but they aren't:
https://github.com/vaadin/flow-components/pull/978/files#diff-255f48ed31d114d33fd62d5f44b260202e622606b1f6ffb8fa39fbbc96306a84R599

Reply:
[Here's the line](https://github.com/vaadin/flow-components/blob/4a61d259fd6b32020243f6f690d7566fd4f97a2a/vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js#L601) where it's needed (referenced in the PR description). I just updated the Flow Components PR to now use the index range but it still requires this change in the Web Component.

Reply:
Yeah, after you updated the Flow Components PR, it is clear. Thanks!